SYCL/ESIMD/api/functional/operators/operator_decrement_and_increment_accuracy_fp_extra.cpp

```diff
@@ -0,0 +1,53 @@
+//==------- operator_logical_not.cpp  - DPC++ ESIMD on-device test ---------==//
```

Comment:
and here as well.

Reply:
this comment has been applied in this commit: 75b472db1fe0777cd84eee7aca25852f5a44d436